core/src/main/java/io/seata/core/lock/Locker.java

```diff
@@ -39,7 +39,25 @@
      * @param rowLock the row lock
      * @return the boolean
      */
-    boolean releaseLock(List<RowLock> rowLock);
+    boolean releaseBranchLock(List<RowLock> rowLock);
+
+    /**
+     * Un lock boolean.
+     *
+     * @param xid the xid
+     * @param branchId the branchId
+     * @return the boolean
+     */
+    boolean releaseBranchLock(String xid, Long branchId);
+
+    /**
+     * Un lock boolean.
+     *
+     * @param xid the xid
+     * @param branchIds the branchIds
+     * @return the boolean
+     */
+    boolean releaseGlobalLock(String xid, List<Long> branchIds);
```

Comment:
how about `boolean releaseBranchLock(String xid, List<Long> branchIds);`

Reply:
another place `LockManager.releaseGlobalSessionLock`

Reply:
> how about `boolean releaseBranchLock(String xid, List<Long> branchIds);`
done.
The original intention is to delete all global locks of GlobalSession, but the interface implementation is more suitable for branch.

Reply:
> another place `LockManager.releaseGlobalSessionLock`
This interface definition is clear and is used to remove GlobalSession Lock, I think it can be retained.